Homework/src/main/java/io/codelex/loops/practice/ExerciseNine.java

```diff
@@ -0,0 +1,19 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class ExerciseNine {
+    public static void main(String[] args) {
+        int dice1 = 0;
+        int dice2 = 0;
+        int sum;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Desired sum:");
+        sum = sc.nextInt();
+        while (dice1 + dice2 != sum) {
+            dice1 = (int) (Math.random() * (6 - 1)) + 1;
```

Comment:
izmanto `Random.nextInt()`, unnecessary complexity un no casting labāk izvairīties pēc iespējas
kā arī šis atkāŗtojas 2x, tādēļ varētu iznes uz papildmetodi `rollDice()` piemēram